pinot-core/src/main/java/org/apache/pinot/core/accounting/PerQueryCPUMemAccountantFactory.java

```diff
@@ -157,6 +160,11 @@ public PerQueryCPUMemResourceUsageAccountant(PinotConfiguration config, String i
       LOGGER.info("_isThreadCPUSamplingEnabled: {}, _isThreadMemorySamplingEnabled: {}", _isThreadCPUSamplingEnabled,
           _isThreadMemorySamplingEnabled);
 
+      _isThreadSamplingEnabledForMSE =
+          config.getProperty(CommonConstants.Accounting.CONFIG_OF_ENABLE_THREAD_SAMPLING_MSE,
+              CommonConstants.Accounting.DEFAULT_ENABLE_THREAD_SAMPLING_MSE);
+      LOGGER.info("_isThreadSamplingEnabledForMSE: {}", _isThreadSamplingEnabledForMSE);
```

Comment:
nit: I don't think using the attribute name is a good idea. Specially in case it changes in the future. Instead something like: `Thread sampling enabled for MSE: true/false`